components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/model/CustomPersistenceEnabledClaims.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.governance.model;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This class holds the custom persistence enabled claims for user store and identity store.
+ */
+public class CustomPersistenceEnabledClaims implements Serializable {
+    private static List<String> userStorePersistentClaims;
```

Comment:
```suggestion
    
    private static List<String> userStorePersistentClaims;
```

Let's format the whole file.